apps/supernova/src/components/silences/SilencesList.tsx

```diff
@@ -43,6 +44,14 @@ const SilencesList = () => {
   const { data, isLoading, error } = useBoundQuery<SilencesData>("silences")
 
   if (error) {
+    // Extra CORS warning based on instanceof
+    if (error instanceof CorsNetworkError) {
+      addMessage({
+        variant: "warning",
+        text: <FirefoxCorsWarning />,
```

Comment:
I think there are other browsers that are built on top of Firefox, IMO the messaging should not be specific to a browser rather it should be around the actual problem, as in, `CorsNetworkError` and in "how to address example" we can then mention that if you are using for example Firefox this is how this can be addressed.

Reply:
The issue just happens in Firefox because it doesn't forward the `allow_client_cert` by default. I just want to bringt back this functionality because it was accidentally removed in the passt. This is a temporary fix until we introduce BFF for greenhouse or supernova itself.